src/features/query-bar/__tests__/QueryBar-test.js

```diff
@@ -20,14 +26,19 @@ describe('features/query-bar/components/QueryBar', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
-    test('should render ColumnButton with columns that do not include item properties', () => {
+    test('should render ColumnButton with columns that does not include property: "name" and source: "metadata"', () => {
```

Comment:
Think of this like documentation. What's a different way to phrase this requirement?